docs/test_framework_integration/junit_5.md

```diff
@@ -1,4 +1,5 @@
-# TestContainers JUnit Jupiter module
+# Jupiter / JUnit 5
```

Comment:
This page contains a very important & generic "Singleton containers" section. 

WDYT about creating "other" under "Test framework integration", putting "Singleton containers" there, and referring it from the JUnit 4 / JUnit 5 pages?

Reply:
That's a good point - will do.